Feature/display users name

avivbarel opened the pull request:
Solves #39

avivbarel committed:
Now display the user's name
Now display the user's full name in the top right corner.
Now instead of "My Files" in sidebar, it's "FirstName's Files".

avivbarel committed:
Refactoring

yonatandt commented on src/utils/auth.js:
```diff
@@ -66,7 +65,7 @@ export async function validateLogin () {
     if (getCookie("kd-token")) {
       parseToken();
     } else {
-      window.open(`/auth/login`, '_self');
```
Don't commit those changes.
We need to find a solution for that.

avivbarel replied:
done

yonatandt commented on src/utils/constants.js:
```diff
@@ -1,6 +1,6 @@
 const name = 'KDrive';
 const disableExternal = false;
-const baseURL = '/';
```
Same

avivbarel replied:
done

yonatandt commented on src/utils/constants.js:
```diff
@@ -13,7 +13,7 @@ const config = {
 };
 
 export async function fetchConfig() {
-	const res = await fetch(`${baseURL}api/config`);
```
same

avivbarel replied:
done

avivbarel committed:
Fixed conflicts

omerzamir committed:
change kartoffel to localhost

Signed-off-by: omerzamir <zamir98@gmail.com>

avivbarel committed:
Temporary fix for env addresses

omerzamir commented on src/components/Sidebar.vue:
```diff
@@ -1,9 +1,9 @@
+      <router-link class="action" to="/files" :aria-label="$t('sidebar.myFiles')" :title="$t('sidebar.myFiles.title')">
         <i class="material-icons">folder</i>
-        <span>{{ $t('sidebar.myFiles') }}</span>
+        <span>{{ $t('sidebar.myFiles.personalized', {person: user.firstName}) }}</span>
```
add condition on personalized to check if exist, if not fallback to title

avivbarel replied:
done

Shahar-Y commented on src/components/Header.vue:
```diff
@@ -183,6 +186,11 @@ export default {
     },
     resetPrompts () {
       this.$store.commit('closeHovers')
+    },
```
Delete the code duplication

Shahar-Y commented on src/components/Header.vue:
```diff
@@ -89,10 +91,11 @@ export default {
   },
   data: function () {
-    return {
+    return {      
```
delete space

avivbarel committed:
Handled cases for empty name
User button now doesn't show if name is empty.
"My files" on sidebar now shows if name is empty.

avivbarel committed:
Merge branch 'develop' into feature/display-users-name

Shahar-Y reviewed (approved):
Approved as far as I'm concerned
